wiki/Content-Style-Guide.md

```diff
@@ -0,0 +1,341 @@
+	name "..."
+	description `...`
+	color ...
+	# The mission's source, destination, and nodes that relate to them:
+	source ...
+	destination ...
+	"distance calculation settings"
+		...
+	clearance ...
+	ignore clearance
+	infiltrating
+	# Mission objectives and nodes that relate to them:
+	deadline ...
+	passengers ...
+	cargo ...
+	blocked "..."
+	illegal ...
+	stealth
+	waypoint ...
+	mark ...
+	stopover ...
+	
+	# How the mission is offered:
+	"offer precedence" ...
+	to offer
+		...
+	to accept
+		...
+	on offer
+		...
+	
+	# Things that occur after the on offer is completed:
+	on accept
+		...
+	on decline
+		...
+	on defer
+		...
+	
+	# NPCs and timers:
+	npc
+		...
+	timer
+		...
+	
+	# on * nodes that occur while in flight.
+	on waypoint
+		...
+	on stopover
+		...
+	on enter
+		...
+	on daily
+		...
+	on disabled
+		...
+	
+	# on * nodes that occur when you complete, try to complete, or fail the mission.
+	to fail
+		...
+	on fail
+		...
+	on abort
+		...
+	on visit
+		...
+	to complete
+		...
+	# The on complete should always come last.
+	on complete
+		...
+```
+
+### `on *` nodes
+
+```html
+on *
+	# Put logs first:
+	log ...
+	# Then any condition changes first:
+	<condition> <assignment>
+	# Then any events:
+	event ...
+	# Then payments:
+	payment ...
+	fine ...
+	debt ...
+	outfit ...
+	(give | take) ship ...
+	# Any miscellaneous actions not mentioned elsewhere can then go next, such as:
+	fail ...
+	mark ...
+	unmark ...
+	music ...
+	message ...
+	# Dialogs and conversations are mutually exclusive, but they should always come last:
+	dialog ...
+	conversation
+		...
+```
+
+### `npc` nodes
+
+```html
+# Objectives have no fixed order
+npc disable "scan cargo" ...
+	# The spawn and despawn conditions
+	to spawn
+		...
+	to despawn
+		...
+	# The fleet's personality, government, and where it spawns.
+	personality ...
+	government ...
+	system ...
+	# Settings for the cargo held by the NPC
+	"cargo settings"
+		...
+	# The ship(s) and/or fleet(s) that make up the NPC.
+	ship ...
+	fleet ...
+	# on * nodes have no fixed order.
+	on *
+		...
+	# The dialog and conversation that should be triggered when NPC's objectives are completed.
+	dialog ...
+	conversation
+		...
+```
+
+## `blocked` dialogs
+
+Any mission which meets the following criteria should have a `blocked` dialog:
+
+1. The mission is not the first mission in a chain (i.e. it's continuing from a previous mission).
+2. The mission does not have the `invisible` tag, has an `on offer` conversation or dialog, or is in any way something that is meant to be visible to the player.
+3. The mission has cargo, passengers, a `to accept` node, or anything else that may cause the mission to fail to offer even if the `on offer` conditions are met.
+
+A mission may contain a `blocked` dialog even if it does not meet these criteria, and an exception can be made for missions which use the `to accept` to wait for another mission to complete first.
```

Comment:
```suggestion
A mission may contain a `blocked` dialog even if it does not meet these criteria, and missions which use the `to accept` to wait for another mission to complete first are exempted if there is no other reason why that mission would fail to offer.
```